CMakeLists.txt

```diff
@@ -307,11 +307,13 @@ if(UNITTEST)
              COMMAND sh -c "cd ${CMAKE_CURRENT_SOURCE_DIR}/octave; SHORT_VERSION_FOR_CTEST=1 octave --no-gui -qf ldpcut.m")
              set_tests_properties(test_CML_ldpcut PROPERTIES PASS_REGULAR_EXPRESSION "Nerr: 0")
 
+    configure_file(octave/run_tnewamp1.m.in octave/run_tnewamp1.m)
+    configure_file(octave/tnewamp1.m.in octave/tnewamp1.m)
     add_test(NAME test_codec2_700c_octave_port
              COMMAND sh -c "cd ${CMAKE_CURRENT_BINARY_DIR}/src;
                             ./c2sim ../../raw/cq_ref.raw --phase0 --postfilter --dump cq_ref --lpc 10 --dump_pitch_e cq_ref_pitche.txt; \
                             cd ${CMAKE_CURRENT_BINARY_DIR}/unittest; ./tnewamp1 ../../raw/cq_ref.raw; \
-                            cd ${CMAKE_CURRENT_SOURCE_DIR}/octave;  \
+                            cd ${CMAKE_CURRENT_BINARY_DIR}/octave;  \
                             octave --no-gui -qf run_tnewamp1.m")
```

Comment:
@tmiw @hobbes1069 can you pls explain how these changes fix the problem?  My CMake knowledge isn't too good !

Reply:
Because run_tnewamp1.m is generated from ru_tnewamp1.m.in using a "configure_file" the output is in the build directory not the source directory.